### ISSUE TITLE
Resize Window event

### ISSUE THREAD

**Issue by AlexInCube** [YoYoGames/GameMaker-Feature-Requests#1221]
### Is your feature request related to a problem?

My interface can adapt to different resolutions, and I want it to adapt when the user changes the window borders. That is, if the user grabs the window by the edges, I want to call for example display_set_gui_width() and gui_resize(). 

### Describe the solution you'd like

Add a Window Resize event to objects in the Other category. 

### Describe alternatives you've considered

You can every step of the game, call window_get_width() and window_get_height(). If they are not equal to the previous values of width and height, then we execute the necessary scaling code.

**Comment by Alphish** [comment]
There is a [Window Resize Draw event](https://manual.yoyogames.com/#t=The_Asset_Editors%2FObject_Properties%2FDraw_Events.htm), but it doesn't seem to work for anything except maybe the UWP-specific event mentioned in the manual. So it would be great to have it handle any kind of resize (the kind of thing that triggers the "Resizing swap chain..." message).

The code I tried putting in the Window Resize event was:
```gml
show_debug_message($"{window_get_width()}x{window_get_height()} ({window_get_fullscreen()})");
```

No message was shown for border-dragging, changing window size with random values passed to `window_set_size(...)` or `window_set_fullscreen(...)` calls, all of which I'd like to handle by the New And Improved Window Resize event. The idea is that the event would fire as soon as new window state becomes stable, making it possible to e.g. correctly resize and position application surface and GUI area (in my earlier resolution-switching systems, I found myself needing Alarms of 5 frames or so).

**Comment by DaveInDev** [comment]
Yes, please make the existing Window Resize event working under all desktop platforms, not only UWP.
